tests/create_certificate/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  test_id = length(data.terraform_remote_state.prereq.outputs) > 0 ? data.terraform_remote_state.prereq.outputs.random_string.result : ""
+
+  zone_name   = "cloudarmor.io"
```

Comment:
should this be a variable? other users might be testing using different `zone_name`

Reply:
I don't think we actually have a way to pass variables into the tests at the moment, with `make test`... I guess using `TF_VAR_<var>` might work?

Reply:
Went ahead and added the var, just with a default so figuring out how to actually pass vars through the test framework can be a future problem....

Reply:
👍